oaknational/Oak-Web-Application
# fix: non existence exit quizes

## Description

Some of the lessons that were released yesterday had no data (intentionally) in their exit quizzes. Unfortunately, the PDFs are generated regardless of whether there is data in the quizzes. This meant the lesson page in OWA knew there wasn’t a quiz because there was no quiz data, but the downloads page, that only checks to see if a PDF exists, does see a PDF so offers it for download. 

Connects up new downloads mv to fix the issue

- List of changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2150--oak-web-application.netlify.thenational.academy/teachers/programmes/english-primary-ks1/units/writing-lower-case-letters-in-print/lessons/introduction-to-skills-and-patterns

There should be no exit quiz in this download

## Checklist
